src/server/scripts/Commands/cs_modify.cpp

```diff
@@ -1021,7 +1021,7 @@ class modify_commandscript : public CommandScript
         if (!*args)
             return false;
 
-        uint16 display_id = (uint16)atoi((char*)args);
+        uint64 display_id = (uint64)atoi((char*)args);
```

Comment:
It's still wrong - atoi does not return values > INT_MAX and this will cause a /W4 warning on msvc about argument truncation from uint64 to uint32 (SetDisplayId accepts uint32)

Change to uint32 and it will be good.

Reply:
It works as off uint64 for me.

Reply:
Changed (probably fixed if any issues occur) uint64 to 32

Reply:
Of course it works, it would just trigger a warning during compilation (and we want to avoid that)